src/components/general/Navbar/Notifica/Notifications/items/EntryReview/index.tsx

```diff
@@ -0,0 +1,213 @@
+            title={projectName}
+            closeModal={closeModal}
+        />,
+    ]), [entriesPageLink, entriesPageLinkWithReview, projectPageLink, projectName, closeModal]);
+
+    const handleNotificationSeenStatusChange = React.useCallback(() => {
+        if (onNotificationSeenStatusChange) {
+            onNotificationSeenStatusChange(
+                notification.id,
+                (notification.status === 'seen' ? 'unseen' : 'seen'),
+            );
+        }
+    }, [notification.id, notification.status, onNotificationSeenStatusChange]);
+
+
+    if (notificationType !== 'entry_review_comment_add'
+            && notificationType !== 'entry_review_comment_modify') {
+        return null;
+    }
+
+
+    let message: React.ReactNode = null;
+    if (notificationType === 'entry_review_comment_add') {
+        switch (notification.data.commentType) {
+            case REVIEW_TYPE_COMMENT:
+                message = _ts('notifications.entryReviewComment', 'commentAdd', {
+                    assigner,
+                    comment: commentLinkItem,
+                });
+                break;
+            case REVIEW_TYPE_APPROVE:
+                message = _ts('notifications.entryReviewComment', 'approve', {
+                    assigner,
+                    entry: commentText ? entryLinkItemWithReview : entryLinkItem,
+                    projectName: projectLinkItem,
+                });
+                break;
+            case REVIEW_TYPE_UNAPPROVE:
+                message = 'bla bla';
+                message = _ts('notifications.entryReviewComment', 'unapprove', {
+                    assigner,
+                    entry: entryLinkItemWithReview,
+                    projectName: projectLinkItem,
+                });
+                break;
+            case REVIEW_TYPE_VERIFY:
+                message = _ts('notifications.entryReviewComment', 'verify', {
+                    assigner,
+                    entry: commentText ? entryLinkItemWithReview : entryLinkItem,
+                    projectName: projectLinkItem,
+                });
+                break;
+            case REVIEW_TYPE_UNVERIFY:
+                message = _ts('notifications.entryReviewComment', 'unverify', {
+                    assigner,
+                    entry: entryLinkItemWithReview,
+                    projectName: projectLinkItem,
+                });
+                break;
+
+            default:
+                message = _ts('notifications.entryReviewComment', 'commentAdd', {
+                    assigner,
+                    comment: commentLinkItem,
+                });
+                break;
+        }
+    } else if (notificationType === 'entry_review_comment_modify') {
+        message = _ts('notifications.entryReviewComment', 'commentUpdate', {
+            comment: commentLinkItem,
+        });
+    }
+
+    return (
+        <Notification
+            notificationId={id}
+            message={message}
```

Comment:
There is a typescript error for the message.

Reply:
Done!